source/MRMesh/MRProgressCallback.h

```diff
@@ -1,5 +1,8 @@
 #pragma once
+#include "MRStringConvert.h"
 #include <functional>
+#include <string>
+#include <filesystem>
```

Comment:
Can we define `getCancelMessage` not here  - you see how many new headers are added. Good place is near `utf8string` function

Reply:
reworked